stable_baselines/trpo_mpi/trpo_mpi.py

```diff
@@ -340,7 +340,7 @@ def fisher_vector_product(vec):
 
 
                         vpredbefore = seg["vpred"]  # predicted value function before update
-                        atarg = (atarg - atarg.mean()) / atarg.std()  # standardized advantage function estimate
+                        atarg = (atarg - atarg.mean()) / (atarg.std() or 1.)  # standardized advantage function estimate
```

Comment:
I'm wondering why this would be more correct than adding an "epsilon" to avoid dividing by zero (which is what is done in the rest of the codebase)

Reply:
In this instance both would have the same effect since we demean. It's up to you: I'm impartial to either.

Reply:
I see, the `atarg` would be zero in both cases. I would prefer an "epsilon" value to be consistent with the rest of the code (and it's easier to understand when reading the code in my opinion).